doc/conf.py

```diff
@@ -163,11 +163,11 @@
     # myst-nb from parsing the .ipynb and .py files generated by sphinx-gallery.
     "gallery/**.ipynb",
     "tutorials/**.ipynb",
-    "get_started/**.ipynb",
+    "intro/**.ipynb",
     "projections/**.ipynb",
     "gallery/**.py",
     "tutorials/**.py",
-    "get_started/**.py",
+    "intro/**.py",
     "projections/**.py",
```

Comment:
Maybe sort them alphabetically or sort based on the order they appear in the doc?

Reply:
> sort based on the order they appear in the doc?

I think this is better. I.e., "intro", "tutorials", "gallery", "projections".

Reply:
See comment https://github.com/GenericMappingTools/pygmt/pull/3571/commits/7ce7855a68735dfba10a8a21e40cf9359667a836.